frontend/src/Components/BottomBar.jsx

```diff
@@ -1,10 +1,21 @@
 import { React } from 'react';
+import PropTypes from 'prop-types';
 // import styles from '../Styles/BottomBar.module.css';
 
-const BottomBar = () => (
+const BottomBar = ({Period, Number}) => (
     <div>
```

Comment:
Not really a problem but you can access period and number directly here (so you dont have to pass props) by using the context
`const { state } = useContext(GameContext)` (check the StatusBar.js component for an example)